### PR TITLE
migration arm/amd jobs to multiarch for 4.15 on azure

### PR DESCRIPTION
**Pull request by lwan-wanglin**
refer https://issues.redhat.com/browse/OCPQE-18165

- non-destructive profile set to arm cluster

- destructive profile set to amd cluster

- for fips, RHEL scale up and other non-arm supported jobs , we still keep it in single-arch file at current phase

- also fix some remaining mixarch jobs which needed to be reduce instances

### CODE REVIEW

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-baselinecaps-vset-arm-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-upi-arm-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-ovn-etcd-encryption-compact-amd-f28-destructive

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-nat-gateway-tp-amd-f28-destructive

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-sno-amd-f28-destructive

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-sno-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-upi-arm-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-sno-amd-f28-destructive

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-sno-arm-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-ovn-etcd-encryption-compact-arm-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-baselinecaps-vset-amd-f28-destructive

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-upi-arm-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-sno-arm-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-ovn-etcd-encryption-compact-arm-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-baselinecaps-vset-amd-f28-destructive

**Comment by lwan-wanglin** [comment]
@jianlinliu @jinyunma the running jobs installation part all succeeded, could you help to review? thanks!

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml [hunk 1]
```diff
@@ -527,6 +527,65 @@ tests:
...
+- as: azure-ipi-fullyprivate-proxy-arm-f14
+  cron: 37 17 10,25 * *
+  steps:
+    cluster_profile: azure-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      E2E_RUN_TAGS: '@arm64 and @azure-ipi and @network-ovnkubernetes and not @fips'
```
Comment (jinyunma) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:583]: compared with amd job, seems missing tag "@proxy"?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml [hunk 2]
```diff
@@ -527,6 +527,65 @@ tests:
...
     workflow: cucushift-installer-rehearse-azure-ipi-disconnected-cco-manual-workload-identity
+- as: azure-ipi-disk-encryption-set-amd-f28-destructive
+  cron: 45 14 13 * *
+  steps:
+    cluster_profile: azure-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @azure-ipi and @network-ovnkubernetes'
```
Comment (jinyunma) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:538]: seems missing tag "not @fips"?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml (hunk 1 above)
Comment (lwan-wanglin) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml]: fixed

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml (hunk 2 above)
Comment (lwan-wanglin) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml]: fixed

**Pull request merged**